index.html

```diff
@@ -0,0 +1,152 @@
+</a>
+        <div class="Account">
+            <select>
+                <option>sign in</option>
+                <option>sign up</option>
+                <option>settings</option>
+            </select>
+        </div>
+        
+
+ 
+    </header>
+    <div class="video-container">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/CQLsdm1ZYAw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+            <div class="video-title-container">
+                <h1 class="video-title">Rema - Calm Down (Official Music Video)</h1>
+                <p class="subscriber-count">
+                    <button class="subscribed">subscribed</button>
+                    <span class="count">2.68M subscribers</span>
+                </p>
+                <div class="icons-container">
+                    <img src="images/icons8-share-48.png" alt="icons8-share-48" class="share-icon">
+                    <img src="images/icons8-cut-50.png" alt="icons8-cut-50" class="clip-icon">
+                    <img src="images/icons8-save-search-24.png" alt="icons8-save-search-24" class="save-icon">
+                    <img src="images/icons8-flag-24.png" alt="icons8-flag-24" class="report-icon">
+                </div>
+                <div class="description">
+                    <div class="video-info">
+                        <h1 class="video-title">Rema - Calm Down (Official Music Video)</h1>
+                        <p class="video-veiws">
+                            <span class="views">319,220,282 </span> views 
+                            <span class="date">Feb 11, 2022 </span> 
+                            <span class="ranking">#7 global top music video </span> 
+                        </p>
+                    <p>Video description</p>
+                    
+                        <p>Stream/Download:<a href="#"> https://rema.lnk.to/RaveandRosesSO</a></p>
+                        <p>Mavin All-Stars Ticket LIVE in LAGOS on DEC 16 🎫 : <a href="#"> https://mavinrecords.link/lfxmavin</a></p></li>
+                        <p>Get Rave & Roses World Tour Tickets: <a href="#">https://heisrema.com/pages/tour   🎫</a></p>
+                    
+                   
+                    <p>Follow Rema</p>
+                    <ul>
+                        <li>TikTok:<a href="#"> https://www.tiktok.com/@badboyrema</a></li>
+                        <li>Instagram:<a href="#"> https://www.instagram.com/heisrema/</a></li>
+                        <li>Facebook:<a href="#"> https://www.facebook.com/heisrema</a></li>
+                        <li>Twitter:<a href="#"> https://twitter.com/heisrema</a></li>
+                        <li>SoundCloud:<a href="#"> https://soundcloud.com/heisrema</a></li>
+                    </ul>

```

Comment:
I would suggest you enclose the following in a <nav> tag. It is used to create a semantic grouping of navigation-related links, which can improve accessibility and make it easier for users to understand the structure and organization of a website.